Makefile

```diff
@@ -61,6 +63,11 @@ clean:
 	rm -rf pennylane_lightning/*_ops*
 	rm -rf *.egg-info
 
+.PHONY: wheel
+wheel:
+	PL_BACKEND=$(PL_BACKEND) python scripts/configure_pyproject_toml.py
```

Comment:
Just for my own knowledge, calling the PL_BACKEND option multiple times will always play nicely? As in, I can go from LQ to LG, to LK back to LQ?

Reply:
Yes, the script will always configure the `pyproject.toml` file for each backend. In any order, any number of times.